mappings/net/minecraft/text/Text.mapping

```diff
@@ -42,6 +42,7 @@ CLASS net/minecraft/class_2561 net/minecraft/text/Text
 		COMMENT <p>A shallow copy is made for the siblings.</p>
 	METHOD method_27662 copy ()Lnet/minecraft/class_5250;
 		COMMENT Copies the text itself, excluding the styles or siblings.
+	METHOD method_30937 getOrderedText ()Lnet/minecraft/class_5481;
```

Comment:
```suggestion
	METHOD method_30937 asOrderedText ()Lnet/minecraft/class_5481;
```
to match `asString`, `asTruncatedString` and similar conversion methods in other classes